packages/strapi-parts/src/Portal/Portal.stories.mdx

```diff
@@ -12,7 +12,15 @@ import { Portal } from './Portal';
 
 This is the doc of the `Portal` component
 
-## Portal
+[View source](https://github.com/strapi/parts/tree/develop/packages/strapi-parts/src/Portal)
+
+## Imports
+
+```js
+import { Portal } from '@strapi/parts/Portal';
```

Comment:
Can you add the PropTypes in the Portal.js file so we can then display them in the story?

Reply:
not sure how to do that. could you show me?